### DISCUSSION TITLE
Version 0.0.6 of PyMuPDF4LLM published

### DISCUSSION THREAD

**Discussion by JorjMcKie** [pymupdf/pymupdf4llm#59] [Announcements]
Version pymupdf4llm 0.0.6 has just been published on [PyPI](https://pypi.org/project/pymupdf4llm/).

Updating the official documentation on ReadTheDocs will take a few days longer.
This version implements significant fixes and feature improvements. So please make sure to have a look at the documentation via this interim link: https://jorjmckie.readthedocs.io/en/latest/.

**Comment by YanSte** [comment]
Hello @JorjMcKie,

Could you please add a tag to the Git repository? This would help us track the latest updates more easily.

Thank you!

**Reply by JorjMcKie** [reply]
Sure, I will!
